### PR TITLE
chore: change @excalidraw/* packages reference

### PR DESCRIPTION
**Pull request by raphaelbs**
- Removes local reference of excalidraw utils (excalidraw-utils-0.1.0.tgz) in favor of using the published version at [@excalidraw/utils](https://www.npmjs.com/package/@excalidraw/utils);
- Changes `@excalidraw/excalidraw` reference to use the npm package: [@excalidraw/excalidraw](https://www.npmjs.com/package/@excalidraw/excalidraw)

### CODE REVIEW

**Comment by dai-shi** [comment]
Hey! Thanks for working on this! Really happy to see someone is interested in contribution.
I just configured vercel. Would you add an empty commit so that it triggers a preview build?

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "homepage": "./",
   "dependencies": {
     "@excalidraw/excalidraw": "npm:aakansha-excalidraw@0.2.1-cdn",
```
Comment (dai-shi) [inline package.json:7]: Would you also try updating the main excalidraw package? I think the cdn asset is already supported.

**Comment by raphaelbs** [comment]
> Hey! Thanks for working on this! Really happy to see someone is interested in contribution.

My pleasure. Building animation on top of Excalidraw was such a great idea, I love it. I intend to push more contributions for sure.
I'll open an issue so we can discuss a roadmap.

> I just configured vercel. Would you add an empty commit so that it triggers a preview build?

Cool! Thanks for doing that. I'll update the PRs once I back to the computer 🖥️.

**Comment by dai-shi** [comment]
@raphaelbs would you merge the master branch (#7 is merged) into this branch?

**Comment by raphaelbs** [comment]
> @raphaelbs would you merge the master branch (#7 is merged) into this branch?

In this particular case, it shouldn't be necessary to update this branch. Either way, I'd rebase it with your master branch.

**Comment by dai-shi** [comment]
> In this particular case, it shouldn't be necessary to update this branch. Either way, I'd rebase it with your master branch.

I know what you mean. Should have clarified the reason. There's an issue I found and wanted to see if #7 solves it.

And it doesn't. Here's the screeenshot of the issue.
image

**Comment by raphaelbs** [comment]
> There's an issue I found and wanted to see if #7 solves it.
> And it doesn't. Here's the screeenshot of the issue.

Gotcha!

What do you think of https://github.com/dai-shi/excalidraw-claymate/pull/10?

**Review by dai-shi — APPROVED** [review]

**Comment by dai-shi** [comment]
Merging this too. I look forward to collaborating with you.

**Pull request merged**